### PR TITLE
Update README for using with gradle/gradle-build-action

### LINKED ISSUE #229: How this action be compitable with gradle/gradle-build-action
My current config:
```yml
      - name: Prepare instrumentation
        uses: reactivecircus/android-emulator-runner@v2
        with:
          api-level: ${{ matrix.api-level }}
          force-avd-creation: false
          emulator-options: -no-snapshot-save -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none
          disable-animations: true
      - name: Instrumentation tests
        uses: gradle/gradle-build-action@v2
        with:
          arguments: connectedDebugAndroidTest
```
Replaced `script: ./gradlew connectedDebugAndroidTest` with executing `connectedDebugAndroidTest` by gradle/gradle-build-action, but error thrown:
```bash
Error: Input required and not supplied: script
```
How can I use this action with `gradle/gradle-build-action`? 
Can reproduce this issue in Goooler/DemoApp/pull/39.

### PR DESCRIPTION
**Pull request by Goooler**
Closes #229.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,6 +138,9 @@ jobs:
           script: ./gradlew connectedCheck
 ```
 
+Note: if you want to use [gradle-build-action](https://github.com/gradle/gradle-build-action) with `reactivecircus/android-emulator-runner`,
+need to declare `uses: gradle/gradle-build-actionv2` without any arguments before `uses: reactivecircus/android-emulator-runner@v2`, more details see [#229](https://github.com/ReactiveCircus/android-emulator-runner/issues/229).
+
```
Comment (ychescale9) [inline README.md:143]: This isn't specific to this action, so I think we can just replace the `actions/cache@v2` step in the snippet above with `uses: gradle/gradle-build-actionv2`.

**Review by ychescale9 — CHANGES_REQUESTED** [review]
Thanks, just 1 change and we're good to go

File: README.md (hunk 1 above)
Comment (Goooler) [reply README.md]: 4fae9cef9371fa74720173ed7919bc20a279476d

Comment (Goooler) [reply README.md]: image

**Pull request merged**